server/risk_scores/risk_assessment.py

```diff
@@ -115,26 +193,20 @@ def get_current_risk_score(form: submit_schema.Form):
                   risk_scores.occurrence_time_to_risk_map.get_risk_score(
                       form.occurrence_time))
 
-    max_risk_score = (
-        risk_scores.program_to_risk_map.max_risk_score() +
-        risk_scores.incident_type_to_risk_map.max_risk_score() +
-        risk_scores.response_to_risk_map.max_risk_score_with_value_count(
-            len(form.immediate_response)) +
-        risk_scores.occurrence_time_to_risk_map.max_risk_score())
-
-    percent_of_max = risk_score / max_risk_score
-    return percent_of_max
+    return normalize_current_risk_score(risk_score)
 
 
-def get_risk_assessment(form: submit_schema.Form, timeframe) -> RiskAssessment:
-    total_risk_score = get_current_risk_score(form)
-    + get_previous_risk_score(form, timeframe)
+def get_risk_assessment(form: submit_schema.Form, timeframe: int) -> RiskAssessment:
+    score_from_current_incident = get_current_risk_score(form) * 0.5
+    score_from_prev_incidents = get_previous_incidents_risk_score(
+        form, timeframe) * 0.5
+    total_risk_score = score_from_current_incident + score_from_prev_incidents
```

Comment:
I'm not sure if we should split the score from current and past incidents equally in half. If an incident gets the maximum possible risk score but doesn't have any previous incidents related to it, it can only get a risk score of 0.5 out of 1. This means it can never be classified as a HIGH risk unless it has some contribution from previous incidents

Reply:
Will be fixed in the PR for NLP-183